### ISSUE TITLE
Crop generation failed ValueError('empty range for randrange)

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#3357]
#### Encountered on:
Linux, source, GOLEM Version: 0.17.1+dev309.g5b3f125

task: Tabasco.blend 400x400 1 frame, 20 subtasks; 15 min for subtask, 1 hour for task

computations for task had been started on two nodes but it seemed that subtasks are stuck on verification. Even though deadline for subtask had passed, status for subtasks had not changed to failure. 

from `golem.log`

```
2018-09-27 12:48:23 INFO     apps.blender.verification           Running verification of subtask 'd5721d7a-c242-11e8-ab54-6c15fa96c125'
2018-09-27 12:48:23 ERROR    apps.blender                        Crop generation failed ValueError('empty range for randrange() (381,360, -21)',)
2018-09-27 12:48:23 WARNING  golem.task.taskmanager              Result for subtask d5721d7a-c242-11e8-ab54-6c15fa96c125 when subtask state is Verifying
```

```
2018-09-27 12:49:10 INFO     golem.network.transport.session     Disconnecting '10.30.11.251':58018 reason: <REASON.NoMoreMessages: 'no_more_messages'>
2018-09-27 12:49:10 INFO     apps.blender.verification           Running verification of subtask 'e63b4fb6-c242-11e8-bcb5-6c15fa96c125'
2018-09-27 12:49:10 ERROR    apps.blender                        Crop generation failed ValueError('empty range for randrange() (361,340, -21)',)
2018-09-27 12:49:10 WARNING  golem.task.taskmanager              Result for subtask e63b4fb6-c242-11e8-bcb5-6c15fa96c125 when subtask state is Verifying
2018-09-27 12:49:11 INFO     golem.task.tasksession              Received offer to compute. task_id='d0895be8-c242-11e8-9ee4-6c15fa96c125', node="'vm win nv'(f420bda8..f1b8af4e)"
2018-09-27 12:49:11 WARNING  golem.task.taskmanager              Waiting for results from f420bda8d39016263852869591efde96c93f13b226e8f8989f325101600c19cc23919bba71197f5f3d71d2ccb56247c775bf027ef547688676588505f1b8af4e on d0895be8-c242-11e8-9ee4-6c15fa96c125
2018-09-27 12:49:11 WARNING  golem.task.tasksession              Can not accept offer: Still waiting on results.task_id='d0895be8-c242-11e8-9ee4-6c15fa96c125', node="'vm win nv'(f420bda8..f1b8af4e)"
```

res for task
https://drive.google.com/open?id=1-Ig1VjE9FbtQ8K8WFbcdaMo51sGtfVE2

**Comment by mfranciszkiewicz** [comment]
I believe we should validate the output crop size within the app during task creation.

**Comment by shadeofblue** [comment]
fix merged into 0.18.0
